tests/mechanic/telemetry_test.py

```diff
@@ -234,11 +239,23 @@ def test_wrong_cluster_name_in_ccr_stats_indices_forbidden(self):
         }
         with self.assertRaisesRegex(exceptions.SystemSetupError,
                                     "The telemetry parameter 'ccr-stats-indices' must be a JSON Object with keys matching the cluster names "
+                                    "\[default,cluster_b\] "
                                     "specified in --target-hosts but it had \[wrong_cluster_name\]."):
-            telemetry.CCRStats(telemetry_params, clients, metrics_store)
+            telemetry.CcrStats(telemetry_params, clients, metrics_store)
+
+
+class CcrStatsRecorderTests(TestCase):
+    @mock.patch("esrally.metrics.EsMetricsStore.put_count_cluster_level")
```

Comment:
Do you need the mock here?

Reply:
Yes we do, test fails without it. It's a good question though as I'd think execution would stop when we are raising `exceptions.RallyError` and the assertion consumes it (and stops there, thus not proceeding in `CcrStatsRecorder()` and reaching `put_value()`).

Reply:
Scratch that. It works fine without mocking of course if I also remove the corresponding parameter in `test_raises_exception_on_transport_error()`

Reply:
Addressed in https://github.com/elastic/rally/pull/505/commits/c9df447ec7fa9339d423ecf9ef6bf34261e15a82